dali/operators/reader/loader/video/frames_decoder_gpu.h

```diff
@@ -108,11 +114,21 @@ class DLL_PUBLIC FramesDecoderGpu : public FramesDecoder {
 
   BufferedFrame& FindEmptySlot();
 
+  bool HasEmptySlot() const;
+
+  bool EmptyBuffer() const;
```

Comment:
Reads like "empty the buffer" == "make it empty".
```suggestion
  bool BufferEmpty() const;
```
or
```suggestion
  bool IsBufferEmpty() const;
```

Reply:
Fixed